frontend/public/models/yaml-templates.ts

```diff
@@ -151,16 +151,15 @@ spec:
     imageChange: {}
   - type: ConfigChange
 `).setIn([referenceForModel(k8sModels.ChargebackReportModel), 'default'], `
-apiVersion: chargeback.coreos.com/v1alpha1
+apiVersion: metering.openshift.io/v1alpha1
 kind: Report
 metadata:
   name: namespace-memory-request
-  namespace: default
+  namespace: metering
```

Comment:
```suggestion
  namespace: openshift-metering
```

Reply:
I can't create a `openshift-metering` namespace?

Reply:
You have to use the namespace dialog, not projects, since it's reserved.

In practice, I don't think ns matter here since it's replaced with the current namespace at runtime in the editor.

Reply:
fixed